src/background/index.ts

```diff
@@ -226,6 +226,7 @@ function getSelectedText(tabId, cb) {
 const messageListener = (request) => {
 
   if (request.message === "auth_user") {
+    console.log(request)
```

Comment:
@joelrojo any idea why the messageListener is not hearing my "auth_user" message from LoginForm.tsx?

Reply:
When i press the button, it's recording the input values correctly but it's not sending it to the backgorund script.

Reply:
I got it to work, might have been the way the values were being passed? But not sure I didn't do much haha. Check out my last commit

Reply:
<img width="1680" alt="Screen Shot 2020-08-03 at 12 53 55 PM" src="https://user-images.githubusercontent.com/217719/89221819-c4aa8700-d588-11ea-8d68-d8903100e752.png">